lighthouse-extension/test/extension-test.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env mocha */
+
+const path = require('path');
+const assert = require('assert');
+const fs = require('fs');
+const puppeteer = require('puppeteer');
+
+const lighthouseExtensionPath = path.resolve(__dirname, '../app');
+const config = require(path.resolve(__dirname, '../../lighthouse-core/config/default.js'));
+const lighthouseCategories = Object.keys(config.categories);
+
+let extensionPage;
+
+const assertAuditElements = async ({category, expected, selector}) => {
```

Comment:
nit: any reason to have these two as arrow functions expressions? e.g. could just be
`async function assertAuditElements({category, expected, selector}) {`

Reply:
no real reason :) whatever you guys prefer

Reply:
arrow fn style makes sense if its a 1liner or 1.5 liner. past that I agree it should get a function declaration.

Reply:
let me memorize this rule for the next time :p